tests/compile.sh

```diff
@@ -48,6 +48,8 @@ if [[ $MACHINE_ID == cheyenne.* ]] ; then
     BUILD_JOBS=${BUILD_JOBS:-3}
 elif [[ $MACHINE_ID == wcoss_dell_p3 ]] ; then
     BUILD_JOBS=${BUILD_JOBS:-1}
+    . $MODULESHOME/init/sh
```

Comment:
Really not sure why this is needed, since the logic in `NEMS/src/conf/module-setup.*` is:
```
else if ( { test -L /usrx && sh -c "readlink /usrx 2> /dev/null | grep dell" } ) then
    # We are on WCOSS Mars or Venus
    if ( ! { module help >& /dev/null } ) then
        source /usrx/local/prod/lmod/lmod/init/$__ms_shell
    endif
    module purge
```

Reply:
I would suggest changing these two lines to: 
source ${PATHTR}/NEMS/src/conf/module-setup.sh.inc